ksql-cli/src/main/java/io/confluent/ksql/cli/console/Console.java

```diff
@@ -377,117 +286,15 @@ private void printAsTable(final GenericRow row) {
-      tableBuilder.withColumnHeaders("Topic Name", "Kafka Topic", "Type");
-      final List<String> topicInfo = new ArrayList<>();
-      final TopicDescription topicDescription = (TopicDescription) ksqlEntity;
-      topicInfo.add(topicDescription.getName());
-      topicInfo.add(topicDescription.getKafkaTopic());
-      topicInfo.add(topicDescription.getFormat());
-      if (topicDescription.getFormat().equalsIgnoreCase("AVRO")) {
-        tableBuilder.withColumnHeaders("AvroSchema");
-        topicInfo.add(topicDescription.getSchemaString());
-      }
-      tableBuilder.withRow(topicInfo);
-    } else if (ksqlEntity instanceof StreamsList) {
-      tableBuilder.withColumnHeaders("Stream Name", "Kafka Topic", "Format");
-      ((StreamsList) ksqlEntity).getStreams().forEach(
-          s -> tableBuilder.withRow(s.getName(), s.getTopic(), s.getFormat()));
-    } else if (ksqlEntity instanceof TablesList) {
-      tableBuilder.withColumnHeaders("Table Name", "Kafka Topic", "Format", "Windowed");
-      ((TablesList) ksqlEntity).getTables().forEach(
-          t -> tableBuilder.withRow(t.getName(), t.getTopic(), t.getFormat(),
-              Boolean.toString(t.getIsWindowed())));
-    } else if (ksqlEntity instanceof KsqlTopicsList) {
-      tableBuilder.withColumnHeaders("Ksql Topic", "Kafka Topic", "Format");
-      ((KsqlTopicsList) ksqlEntity).getTopics().forEach(
-          t -> tableBuilder.withRow(t.getName(), t.getKafkaTopic(), t.getFormat().name()));
-    } else if (ksqlEntity instanceof KafkaTopicsList) {
-      tableBuilder.withColumnHeaders(
-          "Kafka Topic",
-          "Registered",
-          "Partitions",
-          "Partition Replicas",
-          "Consumers",
-          "ConsumerGroups");
-      ((KafkaTopicsList) ksqlEntity).getTopics().forEach(
-          t -> tableBuilder.withRow(
-              t.getName(),
-              Boolean.toString(t.getRegistered()),
-              Integer.toString(t.getReplicaInfo().size()),
-              getTopicReplicaInfo(t.getReplicaInfo()),
-              Integer.toString(t.getConsumerCount()),
-              Integer.toString(t.getConsumerGroupCount())));
-    } else if (ksqlEntity instanceof ExecutionPlan) {
-      final ExecutionPlan executionPlan = (ExecutionPlan) ksqlEntity;
-      tableBuilder.withColumnHeaders("Execution Plan");
-      tableBuilder.withRow(executionPlan.getExecutionPlan());
-    } else if (ksqlEntity instanceof FunctionNameList) {
-      tableBuilder.withColumnHeaders("Function Name", "Type");
-      ((FunctionNameList) ksqlEntity)
-          .getFunctions().stream().sorted()
-          .forEach(func -> tableBuilder.withRow(
-              Arrays.asList(func.getName(), func.getType().name().toUpperCase())));
-    } else if (ksqlEntity instanceof FunctionDescriptionList) {
-      printFunctionDescription((FunctionDescriptionList) ksqlEntity);
-      return;
-    } else {
+    final Handler<KsqlEntity, Console> defaultHandler = (console, entity) -> {
```

Comment:
Do we actually need to have a defaultHandler? i.e, at the moment there is only 1 and the `getOrDefault` method could just be `get` and throw the exception? Doesn't seem like it is really necessary at the moment

Reply:
Yeah, I'd spotted the same.